### ISSUE TITLE
Feature request: t-helper for vue-intl

### ISSUE THREAD

**Issue by fkm** [formatjs/formatjs#3444]
It would be nice to have a t-helper function for code like this:
:heart_eyes: `$t('one')` `$t('two', { foo: 'bar' })` instead of:
:cry:  `$formatMessage({ id: 'one' })` `$formatMessage({ id: 'two' }, { foo: 'bar' })`

Something like this seems to be enough:
`app.config.globalProperties.$t = (id, values) => intl.formatMessage({ id }, values);`

This would be in line with Ember's implementation.
https://ember-intl.github.io/ember-intl/docs/helpers/t

**Comment by glen-84** [comment]
See also https://github.com/formatjs/formatjs/discussions/2476.

**Comment by longlho** [comment]
I'm kinda coming around to this but will have to find time to implement it

**Comment by glen-84** [comment]
Not stale.

**Comment by longlho** [comment]
hmm how do you typically specify `defaultMessage` within vue?

**Comment by glen-84** [comment]
> hmm how do you typically specify `defaultMessage` within vue?

Assuming that nothing has changed since I last used this library (I'm not currently doing f/end):

```vue
{{ $formatMessage({defaultMessage: "All"}) }}
```

**Comment by longlho** [comment]
Right, but the new syntax being proposed with $t seems to be slightly different where 1st arg is the id?

**Comment by glen-84** [comment]
Ya, I had suggested [something similar](https://github.com/formatjs/formatjs/discussions/2476#discussioncomment-260578), which also included the default message.

I don't really like the `"my-id-or-null-for-default"` part though. Perhaps the signature could remain the same, but shorten `defaultMessage` to `default`:

```vue
{{ $fm({default: "default-message"}, {value: 123}) }}
```

Using an ID:

```vue
{{ $fm({id: "one"}, {foo: "bar"}) }}
```

(I prefer `fm` to `t` because you can have `fn` for `formatNumber`, `fd` for `formatDate`, etc.)
